Makefile

```diff
@@ -137,7 +137,7 @@ bin/pulumi-java-gen:
 init_upstream:
 	@if [ ! -f "upstream/.git" ]; then \
 			echo "Initializing upstream submodule" ; \
-			(cd upstream && git submodule update --init && git remote add source git@github.com:hashicorp/terraform-provider-aws.git) ; \
+			(cd upstream && git submodule update --init && git remote add source git@github.com:hashicorp/terraform-provider-aws.git && git fetch source) ; \
```

Comment:
Consider removing the warning "this file is auto-generated" if we're editing it as a source file now?